pgmpy/metrics/metrics.py

```diff
@@ -403,6 +408,15 @@ def fisher_c(model, data, ci_test, show_progress=True):
 
     C = -2 * np.log(cis.loc[:, "p_value"]).sum()
     p_value = 1 - stats.chi2.cdf(C, df=2 * cis.shape[0])
+    rmsea = np.nan
+
+    if calculate_rmsea:
+        if len(data) != 1 and len(cis) != 0:
+            rmsea = np.sqrt(
+                max((C - 2 * len(cis)) / (2 * len(cis) * (len(data) - 1)), 0)
+            )
+        return (p_value, rmsea)
```

Comment:
We will need to update the `Returns` section in the docstring to reflect this change.